config/routes.rb

```diff
@@ -3,6 +3,7 @@
   match 'approval_response', to: 'communicarts#approval_response', via: [:get, :put]
   root :to => 'home#index'
   match "/auth/:provider/callback" => "home#oauth_callback", via: [:get]
+  get '/error' => 'home#error'
```

Comment:
No, I think it's useful in various environments for testing things like New Relic and logging.